lib/MBTA.rb

```diff
@@ -0,0 +1,52 @@
+class Trip
+  MBTA = {
+    "Red" => [
+      "Alewife", "Davis", "Porter", "Harvard", "Central", "Kendall/MIT", "Park Street", "South Station"
+    ],
+    "Green" => [
+      "Haymarket", "Government Center", "Park Street", "Boylston", "Arlington", "Copley"
+    ],
+    "Orange" => [
+      "North Station", "Haymarket", "Park Street", "State Street", "Downtown Crossing", "Chinatown", "Tufts Medical Center"
+    ]
+  }
+
+  def initialize(oriLine:, oriStop:, destLine:, destStop:)
+    @oriLine = oriLine
+    @oriStop = oriStop
+    @destLine = destLine
+    @destStop = destStop
+  end
+
+  def oriLine
+    @oriLine
+  end
+
+  def oriStop
+    @oriStop
+  end
+
+  def destLine
+    @destLine
+  end
+
+  def destStop
+    @destStop
+  end
+
+  def transfer?
+    oriLine == destLine
```

Comment:
I believe your method is exactly the opposite of what it should be. Wouldn't `oriLine != destLine` make more sense here?